client/src/components/organisms/DonorImpactSection.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { FunctionComponent, useState } from "react";
+import Col from 'react-bootstrap/Col'
+import Row from 'react-bootstrap/Row'
+
+import MapWithMarkers, { MapWithMarkersPoint } from '../atoms/MapWithMarkers'
+import { Statistic, Testimonial } from '../../data/types/donorHomepageConfig'
+import BaseMap from '../../assets/kw-region-map.png'
+import Card from '../atoms/CardWithShadow'
+import Cursor from '../../assets/cursor.png'
+import DonorHomepageConfig from '../../config/donorHompageConfig.json'
+
+const DonorImpactSection: FunctionComponent = () => {
+  const [selectedTestimonial, _setSelectedTestimonial] = useState<Testimonial | null>(null);
+  const testimonials = DonorHomepageConfig.map.testimonials;
+  const setSelectedTestimonial = (id: number | null) => {
+    if (!id) {
+      _setSelectedTestimonial(null);
+      return;
+    }
+    const testimonial = testimonials.find((testimonial: Testimonial) => testimonial.id === id);
+    _setSelectedTestimonial(testimonial ?? null);
+  }
+  const points = DonorHomepageConfig.map.points;
+  const markerSize = testimonials.length > 0 && testimonials.length <= DonorHomepageConfig.map.markerSizes.length
```

Comment:
Isn't this selecting only one marker size when multiple should be used? Maybe I'm confused as to why we distinguish multiple marker sizes.

Reply:
Designers want to marker size to change depending on the amount of markers
Less markers => larger marker size
More markers => smaller marker size

See description screenshots to see how marker sizes change with the number of markers